kirin/utils.py

```diff
@@ -128,7 +129,7 @@ def should_retry_exception(exception):
 def make_kirin_lock_name(*args):
     from kirin import app
 
-    return "|".join([app.config["TASK_LOCK_PREFIX"]] + [str(a) for a in args])
+    return "|".join([app.config[str("TASK_LOCK_PREFIX")]] + [str(a) for a in args])
```

Comment:
```suggestion
    return "|".join([app.config[str("TASK_LOCK_PREFIX")]] + [six.text_type(a) for a in args])
```
there is probably others `str()` that need to be changed